content-api/content-service/conf/application.conf

```diff
@@ -736,3 +736,6 @@ collection {
 		}
 	}
 }
+
+#Request timeout
+actor.timeoutMillisec = 120000 
```

Comment:
Keep the default value 30000L. In your env, you can configure it 120000L in config file.